renovate.json

```
@@ -11,6 +11,9 @@
   ],
   "automergeStrategy": "squash",
   "automergeType": "pr",
+  "schedule": [
+    "* * * * 0,1"
```

```suggestion
    "* 0-3 * * 1"
```
lets set it to early morning on mondays